.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-exclude=.git,deps,lib,src,test/fixtures,tools/*_macros.py,tools/gyp,tools/inspector_protocol,tools/jinja2,tools/markupsafe,tools/pip,tools/v8_gypfiles/broken
-select=E901,E999,F821,F822,F823
+exclude=.git,deps,lib,src,tools/gyp,tools/inspector_protocol,tools/pip,tools/v8_gypfiles/broken
+select=E9,F82
```

Comment:
OPTIONAL but can we try tightening the screws one more time with: __select=E9,F63,F72,F82__

Reply:
I got only 8 errors with `select=E9,F63,F72,F82`. I might give it a go.

Reply:
Went for the maximally reasonable set